python/ray/llm/_internal/batch/stages/base.py

```diff
@@ -73,6 +73,10 @@ class StatefulStageUDF:
         column for the next stage.
     """
 
+    # The internal column name for the index of the row in the batch.
+    # This is used to align the output of the UDF with the input batch.
+    idx_in_batch_column: str = "__idx_in_batch"
```

Comment:
```suggestion
    IDX_IN_BATCH_COLUMN: str = "__idx_in_batch"
```
nit, upper case for constants.